### PR TITLE
Setup for initial vespa indexing tests

### PR DESCRIPTION
**Pull request by olaughter**
This adds a vespa indexing test. The idea is to have a high level integration test to make it easier to do further work (like refactoring, bug fixes). This test currently just confirms the command still runs successfully with a small amount of local data.

### CODE REVIEW

File: src/config.py [hunk 1]
```diff
@@ -66,3 +66,4 @@ def _convert_to_bool(x: str) -> bool:
 VESPA_CERT_LOCATION: str = os.getenv("VESPA_CERT_LOCATION", "")
 VESPA_KEY_LOCATION: str = os.getenv("VESPA_KEY_LOCATION", "")
 VESPA_NAMESPACE_PREFIX: str = os.getenv("VESPA_NAMESPACE_PREFIX", "navigator")
+DEV_MODE: bool = os.getenv("DEV_MODE", "False").lower() == "true"
```
Comment (katybaulch) [inline src/config.py:69]: Think this should be `DEVELOPMENT_MODE` not `DEV_MODE`

File: tests/test_integration.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    result = runner.invoke(
+        run_as_cli,
+        args = [
+            fixture_dir,
+            "--index-type",
+            "vespa",
+        ],
+        env={
+            "VESPA_INSTANCE_URL": "http://localhost:8080/",
+            "DEV_MODE": "True"
```
Comment (katybaulch) [inline tests/test_integration.py:20]: May wish to rename this for consistency

**Review by katybaulch — CHANGES_REQUESTED** [review]
Eyyy great start on this! All looking good, just need to reference 'DEVELOPMENT_MODE' rather than 'DEV_MODE' for the env variable as that is what the env var in the backend is called (unless there's a separate Vespa DEV_MODE variable I've missed)

File: src/config.py (hunk 1 above)
Comment (olaughter) [reply src/config.py]: Good shout. Fixed

**Review by katybaulch — APPROVED** [review]
Thanks for making that change Fred! Approved

**Pull request merged**